Makefile

```diff
@@ -207,6 +217,11 @@ local-uninstall-chrome:
 	rm -f ~/.config/google-chrome/NativeMessagingHosts/linux_entra_sso.json ~/.config/google-chrome/linux-entra-sso.py
 	rm -f ~/.config/chromium/NativeMessagingHosts/linux_entra_sso.json
 
-local-uninstall: local-uninstall-firefox local-uninstall-chrome
+local-uninstall-brave:
+	rm -f ~/.config/BraveSoftware/Brave-Browser/NativeMessagingHosts/linux_entra_sso.json ~/.config/google-chrome/linux-entra-sso.py
+
+local-uninstall: local-uninstall-firefox local-uninstall-chrome local-uninstall-brave
 
-.PHONY: clean release local-install-firefox local-install-chrome local-install install local-uninstall-firefox local-uninstall-chrome local-uninstall uninstall deb deb_clean
+.PHONY: clean release local-install-firefox local-install-chrome local-install
+.PHONY: install local-uninstall-firefox local-uninstall-chrome local-uninstall uninstall deb deb_clean
+.PHONY: local-install-brave local-uninstall-brave
```

Comment:
If we already split this, can we please sort them by target (e.g. install, uninstall, remaining)?